src/views/open-api-docs-view/types.ts

```diff
@@ -72,6 +72,27 @@ export interface OpenApiDocsParams extends ParsedUrlQuery {
 	page: string[]
 }
 
+/**
+ * Nav items are used to render the sidebar and mobile nav.
+ *
+ * TODO: move these types to sidebar component.
+ * For now, this is difficult, as the MenuItem type is a complex
+ * interface that requires a larger effort to untangle, and all
+ * related Sidebar components are similarly entangled.
+ * Rationale is to start with simpler slightly duplicative types here,
+ * rather than try to embark on the `MenuItem` type refactor.
+ * Task: https://app.asana.com/0/1202097197789424/1202405210286689/f
```

Comment:
I started down the path of trying to unify these types with the component that ultimately renders them (`SidebarNavMenuItem`), but the complex `MenuItem` type struck again, and after spending a couple minutes I felt the scope of untangling that type merits a separate chunk of work.

I really wanna get that `MenuItem` refactor work queued up as the next backlog thing to work on. So I assigned myself 🎟 [Clean up references to MenuItem interface](https://app.asana.com/0/1202097197789424/1202405210286689/f), and my plan is to pick it up next "Backlog Friday" 👍

Reply:
`MenuItem`! Dangit. Awesome, sounds like a good plan.